### PR TITLE
Add Windows import libs.

### PR DESCRIPTION
**Pull request by rjframe**
I've added the import libraries for Windows.

I also added a makefile (for Digital Mars make) to generate them from the DLLs. The makefile requires using the respective x86/x64 Visual Studio Native Tools command prompt and specifying the build as "make x86" or "make x64".

We could build both import libs at once by locating vcvarsall.bat, but since that depends on the VS version, it didn't seem like a reliable option; at least for now.

### CODE REVIEW

**Comment by biozic** [comment]
Thanks! I don't use Windows, so your help is appreciated :)

A `sqlite3.def` file is provided along `sqlite3.dll` when you download the precompiles binaries for Windows from the SQLite website. I just have to add the `.def` file to the package. Then you would only need to build the `.lib` file in your Makefile with the `lib` command.

**Comment by rjframe** [comment]
That works so much better; lib can generate both 32- and 64-bit import libs at once.

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,6 @@ TODO.txt
 docs.json
 docs
 sqlite3.o
+lib/*/*.txt
+*.def
+*.exp
```
Comment (biozic) [inline .gitignore]: Only `lib/*/*.lib` is needed now.

**Comment by biozic** [comment]
Great. Can you update the `.gitignore` file and remove the `.lib` files?

**Comment by rjframe** [comment]
The .lib files are needed for the `all-included` configuration to work on Windows; I can look for a way to identify lib's location to build them on-demand, but since the path will vary depending on the VS version installed, I don't know how reliable it would/could be.

**Comment by biozic** [comment]
> The .lib files are needed for the `all-included` configuration to work on Windows;

I understand that.

> I can look for a way to identify lib's location to build them on-demand, but since the path will vary depending on the VS version installed, I don't know how reliable it would/could be.

I wish there were an easy and reliable way to call `lib` to generate the `lib` files, but if you say it's risky, I'm OK to include them.

What about `.gitignore`?

**Comment by rjframe** [comment]
@biozic Done.

**Pull request merged**